docker-java-core/src/main/java/com/github/dockerjava/core/AbstractDockerCmdExecFactory.java

```diff
@@ -150,6 +150,9 @@
 
     private DockerClientConfig dockerClientConfig;
 
+    protected Integer connectTimeout;
+    protected Integer readTimeout;
```

Comment:
`readTimeout` is wrong since there can be long living connections. The correct approach would be to use timeouts/deadlines when executing the commands

Reply:
For library that doesn't matter, set any timeouts on your app side.

Reply:
you should not be setting the default read timeout here. Some operations last minutes or even hours. The callee should set the timeouts at the use site